opm/simulators/flow/SimulatorFullyImplicitBlackoilEbos.cpp

```diff
@@ -61,4 +65,63 @@ void outputTimestampFIP(const SimulatorTimer& timer,
     OpmLog::note(ss.str());
 }
 
+void checkSerializedCmdLine(const std::string& current,
+                            const std::string& stored)
+{
+    auto filter_strings = [](const std::vector<std::string>& input)
+    {
+        std::vector<std::string> output;
+        output.reserve(input.size());
+        std::copy_if(input.begin(), input.end(), std::back_inserter(output),
+                     [](const std::string& line)
+                     {
+                        return line.compare(0, 11, "EclDeckFile") != 0 &&
+                               line.compare(0, 8, "LoadStep") != 0 &&
+                               line.compare(0, 8, "SaveFile") != 0 &&
+                               line.compare(0, 8, "SaveStep") != 0;
```

Comment:
Any reason not to use `line != ...` here?

Reply:
yes. ideally it would be .startswith() but that's c++20. those params have values, i want to ignore these lines no matter their values.